src/platform.h.in

```diff
@@ -240,4 +240,18 @@ private:
 #define NCNN_LOGE(...)
 #endif
 
+#ifdef _MSC_VER
+    #define forceinline __forceinline
+#elif defined(__GNUC__)
+    #define forceinline inline __attribute__((__always_inline__))
+#elif defined(__CLANG__)
+    #if __has_attribute(__always_inline__)
+        #define forceinline inline __attribute__((__always_inline__))
+    #else
+        #define forceinline inline
+    #endif
+#else
+    #define forceinline inline
+#endif
+
```

Comment:
the forceinline name may conflict with other third-party libraries or user-defined macros
what about NCNN_FORCEINLINE ?